sous/Starter.php

```diff
@@ -22,15 +22,15 @@ public static function installTheme() {
   $drupalFinder->locateRoot(getcwd());
   $composerRoot = str_replace('-', '_', strtolower(basename($drupalFinder->getComposerRoot())));
   // Execute the Emulsify theme build based on composer create path.
-  shell_exec ("cd web/themes/contrib/emulsify-design-system/ && php emulsify.php $composerRoot");
-  shell_exec ("cd web/themes/contrib/emulsify-design-system/ && npm install");
+  shell_exec ("cd web/themes/contrib/emulsify-design-system/ && php emulsify.php emulsify");
```

Comment:
Making this change is taking away the custom theme name for the project. Not sure we want to go that route.

Reply:
**Background:**
When this was originally setup the upstream theme was called `emulsify` but now we have two called:

`emulsify-design-system`
`emulsify-drupal`

So technically the `emulsify` name is now available.

**Old logic problems:**
As an example on the site I am working on now the directory was named `texaslawhelp.org`, so this script would have tried to create a theme called `texaslawhelp.org`

Brian ended up naming the theme on our project `tlh_emulsify`, but I am trying to see the benefit of naming every project we work on something different.

**New logic problems:**
I agree that picking a name for the end developer isn't an ideal situation though either.

**Ideal world:**
Ideally we would allow the developer to setting up the project to pick name the theme but that would require altering the rebuild.sh file with a different theme name for every project.  Not a terrible ask but does require some additional work.

Reply:
My recommendation would be that we change it back. That was/is one of the main points of this starter. To have a custom theme generated from Emulsify.

I see what you mean about that needing that consistency for the purpose of the shell build tool but I don't think that script should be the catalyst for this change. Maybe we consider this `build-theme.sh` file also gets generated on the fly. We can have a script that creates it fo us using the same method we are using to create the theme. OR we have some kind of documentation to address the need.

Also I found one more thing. There has been a recent change to the emulsify package name, which you mentioned. That has caused the contrib directory to change so this line should look like this:

`shell_exec ("cd web/themes/contrib/emulsify-drupal/ && php emulsify.php $composerRoot");`

Reply:
I am thinking we should put this PR on hold because the emulsify.php script is going to be replaced with the new Emulsify CLI soon.